tardis/energy_input/indivisible_packets.py

```diff
@@ -295,7 +361,7 @@ def main_gamma_ray_loop(
     photoabsorption_opacity="tardis",
     pair_creation_opacity="tardis",
     seed=1,
-    path_to_artis_lines="",
+    path_to_decay_data="",
```

Comment:
is `path_to_decay_data` the file that is hardcoded in `tardis/energy_input/energy_source.py` line 259?

Reply:
Yes, the user can override the path with this kwarg